ceph_deploy/hosts/centos/install.py

```diff
@@ -198,11 +200,8 @@ def repo_install(distro, reponame, baseurl, gpgkey, **kw):
 
     # set the right priority
     if kw.get('priority'):
-        distro.packager.install_priorities_plugin()
-        logger.warning(
-            'ensuring that {repo_path} contains a high priority'.format(
-                repo_path=repo_path)
-        )
+        if isinstance(distro.packager, Yum):
```

Comment:
I didn't mean it was bad practice, if you don't foresee checking the name of the package manager often then this would work fine.

If this is not a one-off, then you are forcing an import of the class to check. Since this is a class you control, adding an attribute to help these types of checks would be nice.